local-setup/scripts/enable-dashboard.sh

```
@@ -0,0 +1,1193 @@
+  #
+  # Verify against the locales the deployment actually OFFERS (StateInfo
+  # .languages), not just the ones we ship packs for. A locale a user can pick
+  # from the language menu but that we never seeded is precisely the gap that
+  # renders raw DASHBOARD_* keys — checking only our own packs would call that
+  # deployment clean. (Live example: bomet offers fr_FR, which has no repo pack.)
+  local locales="${DASHBOARD_LOCALES//,/ }"
+  if [[ -z "$locales" ]]; then
+    locales="$(psql_q "select string_agg(distinct l->>'value', ' ') from eg_mdms_data d,
+        lateral jsonb_array_elements(d.data->'languages') l
+        where d.schemacode='common-masters.StateInfo' and d.tenantid='$DASHBOARD_TENANT'")"
+    [[ -n "$locales" ]] || locales="$(cd "$DSS_L10N_DIR" && ls ./*.json | sed 's|.*/||; s/\.json$//' | tr '\n' ' ')"
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)